rules/windows/defense_evasion_defendercontrol_activity.toml

```
@@ -0,0 +1,56 @@
+[metadata]
+creation_date = "2020/12/23"
+maturity = "production"
+updated_date = "2021/1/26"
+
+[rule]
+author = ["Elastic"]
+description = "DefenderControl is a portable utility used to disable Windows Defender through the use of the command-line or a GUI."
+false_positives = [
+    "DefenderControl is a legitimate tool that could be used by administrators to perform specific functions.",
+]
+from = "now-9m"
+index = ["logs-endpoint.events.*"]
+language = "eql"
+license = "Elastic License"
+name = "DefenderControl Activity Detected"
+note = "Detections should be investigated to identify if the hosts and users are authorized to use this tool. As this rule detects post-exploitation process activity, investigations into this should be prioritized"
+references = ["https://thedfirreport.com/2020/12/13/defender-control/"]
+risk_score = 21
+rule_id = "2ffa1f1e-b6db-47fa-994b-1512743847eb"
+severity = "low"
+tags = ["Elastic", "Host", "Windows", "Threat Detection", "Defense Evasion"]
+type = "eql"
+
+query = '''
+registry where event.type in ("creation", "change") and
+  ((registry.path:"HKLM\\SOFTWARE\\Policies\\Microsoft\\Windows Defender\\DisableAntiSpyware" and
+  registry.data.strings:"1") or
+  (registry.path:"HKLM\\System\\ControlSet*\\Services\\WinDefend\\Start" and
+  registry.data.strings:"3"))
```

nit (for more obvious precedence and readability):

```suggestion
registry where event.type in ("creation", "change") and
  ((registry.path:"HKLM\\SOFTWARE\\Policies\\Microsoft\\Windows Defender\\DisableAntiSpyware" and
       registry.data.strings:"1") or
   (registry.path:"HKLM\\System\\ControlSet*\\Services\\WinDefend\\Start" and
      registry.data.strings:"3"))
```